roles/edpm_pre_adoption_validation/molecule/kernel-args-existing-args/molecule.yml

```diff
@@ -0,0 +1,37 @@
+---
+dependency:
+  name: galaxy
+driver:
+  name: podman
+platforms:
+- name: edpm-0
+  command: /sbin/init
+  dockerfile: ../../../../molecule/common/Containerfile.j2
+  image: ${EDPM_ANSIBLE_MOLECULE_IMAGE:-"ubi9/ubi-init"}
+  registry:
+    url: ${EDPM_ANSIBLE_MOLECULE_REGISTRY:-"registry.access.redhat.com"}
+  ulimits:
+  - host
+provisioner:
+  name: ansible
+  inventory:
+    hosts:
+      all:
+        vars:
+          edpm_pre_adoption_validation_hostname_enabled: false
+          # this is a kernel arg that by default defined in the test env
+          # we use this to simulate that what is requested during adoption
+          # is already configured
+          edpm_kernel_args: "net.ifnames=0"
```

Comment:
and it turned out that the github runner uses different images with different cmdline than my local env.